packages/coderskit/src/atoms/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { HTMLAttributes } from 'react';
+import styled from '@emotion/styled';
+import classnames from 'classnames';
+
+export type AvatarVariant = 'photo' | 'text';
+export type AvatarShape = 'circle' | 'square';
+
+export interface AvatarProps extends HTMLAttributes<any> {
+  variant?: AvatarVariant;
+  size?: number;
+  shape?: AvatarShape;
+  image?: string;
+  children?: React.ReactNode;
+}
+
+const AvatarBase = styled.div<AvatarProps>(props => {
+  const { shadows } = props.theme;
+  const { size, shape } = props;
+
+  const calcRadius = () => {
+    if (shape === 'circle') {
+      return '100%';
+    } else if (size === 1 || size === 2) {
+      return '4px';
+    } else {
+      // handles size === 3, size === 4 and any other size
+      return '8px';
+    }
+  };
+
+  const calcDimensions = () => {
```

Comment:
Tutaj moja wina, bo tego nie opisałem. Generalnie to aby zachować większą elastyczność tego komponentu, to lepiej żeby otrzymywał on property `size` jako dowolny number i na tej podstawie definiował wysokość i szerokość. W takiej sytuacji w `calcRadius` możesz napisać, że jeśli `size >= 32` to zwraca `8px` a jak nie to `4px`

Reply:
Przepisane. Teraz można podać dowolny number jako size, na tej podstawie dopasowuje się font size i reszta properties. Jest tylko taki problem, że storybook pozwala na nulla w polu gdzie powinien być size i wtedy nie aplikują się default propsy (czyli jak usuniesz wartość size, to komponent nie będzie miał żadnego rozmiaru).